### DISCUSSION TITLE
Maintainer needed

### DISCUSSION THREAD

**Discussion by Nama** [Nama/sway-workspaces#13] [Announcements]
I don't use sway anymore, I can't update this project anymore. I can transfer this project to someone willing to maintain it.

I didn't expected so many people using this project. So I don't want to just archive and be done with it.
There weren't much contributors so I have no idea who could do it. @mishurov recently [commented](https://github.com/swaywm/sway/issues/1005#issuecomment-2125595790) with his own solution and lib. Maybe you?

I still don't think that swayrst should start applications and that i3ipc-python is bloated, but you could do it your way then.

**Comment by mishurov** [comment]
@Nama no, I have no time for that. I made a script for my personal daily use. I just shared it because I though you were still supporting the project and could be interested.

**Comment by Nama** [comment]
am back i guess
